src/test/java/teammates/test/pageobjects/GoogleLoginPage.java

```diff
@@ -12,7 +12,7 @@
 
 public class GoogleLoginPage extends LoginPage {
 
-    private static final String EXPECTED_SNIPPET_SIGN_IN = "Sign in - Google Accounts";
+    private static final String EXPECTED_SNIPPET_SIGN_IN = "Sign in - Google accounts";
```

Comment:
`-` should be changed to `–` :P

After that, @tanhengyeow you may want to help to try the fix with your own staging server.